### ISSUE TITLE
Express learning rates relatively

### ISSUE THREAD

**Issue by tbekolay** [nengo/nengo#555]
Had a brainstorm while finishing up #403. Right now, the learning rates used in `PES`, `Oja`, and `BCM` are pretty mysterious; in each example and test I tweak them in weird ways that feel weird.

I think it could be somewhat less weird if we allow learning rates to be set relative to magic constant that represents a "normal" learning rate for that particular learning rule. Then, instead of setting the learning rate to `1e-6` or whatever, you would set it to a number that will be multiplied by the normal learning rate. So, if you want to learn twice as quick as "normal", you'd set `learning_rate=2` and if you want it to learn half as quick as "normal" `learning_rate=0.5`.

Does this make sense? I feel like it makes learning rules a bit less magical and easy to work with from the user perspective, but there is the danger that people will incorrectly report learning rates in papers because they don't realize it's a relative thing. Perhaps we can make this more clear through the API somehow.

Also, relatedly, we need a good way to determine what the "normal" learning rate should be. While we can just define some ballpark value from our experience for the time being, I think it would be better to come up with metrics to determine a good learning rate, though that metric might have to be learning rule specific... E.g., for PES, it could be the rate at which a connection with a specific weight matrix changes to weights that transmit a static scalar faithfully in 1 second. Or something like that. Seems arbitrary, but all of these things are going to be arbitrary, so might as well pick something and heavily document it rather than tweaking by hand every time.
